docs/MESSAGING.md

```diff
@@ -144,6 +153,7 @@ parameters/data when sending the push notification, you need to set the `showNot
 
 ```js
 firebase.init({
+  // ..
```

Comment:
Just added these because I had the impression this might be all I need to get things working

Reply:
I'm not sure what you mean.. can you please explain?

Reply:
Sorry, what I meant was that when I saw just:
```
firebase.init({
  showNotifications: true,
});
```
Without the `// ...` I thought that maybe that was all I needed to get things working.  The `// ...` implies there are other things that have to be in there. Maybe that's just me but having just this flag won't really do anything 🤷‍♂

Reply:
Ah, gotcha. Yep you're right. Makes sense.